src/pipecat/transports/services/helpers/daily_rest.py

```diff
@@ -33,6 +33,14 @@ class DailyRoomSipParams(BaseModel):
     num_endpoints: int = 1
 
 
+class RecordingsBucketConfig(BaseModel):
+    # https://docs.daily.co/guides/products/live-streaming-recording/storing-recordings-in-a-custom-s3-bucket
```

Comment:
Maybe a docstring in place of this comment?

```suggestion
   """Configuration for storing Daily recordings in a custom S3 bucket.
    
    Refer to the Daily API documentation for more information:
    https://docs.daily.co/guides/products/live-streaming-recording/storing-recordings-in-a-custom-s3-bucket
    """```